src/controllers/user/index.js

```diff
@@ -0,0 +1,11 @@
+
+export const currentUser = (obj, args, { currentUser }) => currentUser
+
+export const groupsSubscribed = async (obj, args, { models }) => {
```

Comment:
This could be resolved easier with a relationship on Sequelize:

User.belongsTo(group, { through: GroupSubscription })

Reply:
Fixed